lib/Module/Build.pm

```diff
@@ -982,71 +981,54 @@ ExtUtils::MakeMaker> for further information.
-C<WriteMakefile()> is actually blessed into a package name that's
-created on the fly, so you can't simply subclass
-C<ExtUtils::MakeMaker>.  There is a workaround C<MY> package that lets
-you override certain C<MakeMaker> methods, but only certain explicitly
-preselected (by C<MakeMaker>) methods can be overridden.  Also, the method
-of customization is very crude: you have to modify a string containing
-the Makefile text for the particular target.  Since these strings
-aren't documented, and I<can't> be documented (they take on different
-values depending on the platform, version of perl, version of
-C<MakeMaker>, etc.), you have no guarantee that your modifications will
-work on someone else's machine or after an upgrade of C<MakeMaker> or
-perl.
+L<ExtUtils::MakeMaker> has been a core module in every version of Perl 5, and
+must maintain compatibility to install the majority of CPAN modules.
+C<Module::Build> was added to core in Perl 5.10 and removed from core in Perl
+5.20, and (like L<ExtUtils::MakeMaker>) is only updated to fix critical issues
+and maintain compatibility. C<Module::Build> and other non-core installers like
+L<Module::Build::Tiny> are installed from CPAN by declaring themselves as a
+C<configure> phase prerequisite, and in this way any installer can be used in
+place of L<ExtUtils::MakeMaker>.
 
 =item *
 
-It is risky to make major changes to C<MakeMaker>, since it does so many
-things, is so important, and generally works.  C<Module::Build> is an
-entirely separate package so that I can work on it all I want, without
-worrying about backward compatibility with C<MakeMaker>.
+Customizing the build process with L<ExtUtils::MakeMaker> involves overriding
+certain methods that form the F<Makefile> by defining the subs in the C<MY::>
+namespace, requiring in-depth knowledge of F<Makefile>, but allowing targeted
+customization of the entire build. Customizing C<Module::Build> involves
+subclassing C<Module::Build> itself, adding or overriding pure-perl methods
+that represent build actions, which are invoked as arguments passed to the
+generated C<./Build> script. This is a simpler concept but requires redefining
+the standard build actions to invoke your customizations, and makes it more
+difficult to combine different customizations for the same action.
+L<Module::Build::Tiny> does not allow for customization.
 
 =item *
 
-Finally, Perl is said to be a language for system administration.
-Could it really be the case that Perl isn't up to the task of building
-and installing software?  Even if that software is a bunch of
-C<.pm> files that just need to be copied from one place to
-another?  My sense was that we could design a system to accomplish
-this in a flexible, extensible, and friendly manner.  Or die trying.
+C<Module::Build> provides more features and a better experience for distribution
+authors than L<ExtUtils::MakeMaker>. However, tools designed specifically for
```

Comment:
I feel like the extending experience is covered well enough by the previous point. This is more referring to M::B's slightly better-designed API and less required boilerplate for everyday use, and the additional author-focused actions (not having authored any M::B dists, I'm not sure how helpful they are). Though you're right it is overall not that different.